extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +138,466 @@ For example
+* Data type and bit depth of the image must be compatible with the property type.
+
+Consequently, an 8-bit per pixel RGB image is only compatible with `UINT8` or normalized `UINT8` properties, and array properties thereof with three components or fewer.
+
+> **Implementation note:** Use of floating-point properties in a property texture would require a floating point-compatible image format like KTX2 provided by an additional extension.
+
+> **Example:** Property texture implementing a "thermalSample" class, with property values stored in a glTF texture at index 0 and indexed by `TEXCOORD_0`.
+>
+> <img src="figures/feature-texture.png"  alt="Property Texture" width="500">
+>
+> _Class and property texture_
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": {
+>         "classes": {
+>           "thermalSample": {
+>             "name": "Thermal Texture",
+>             "properties": {
+>               "thermalTemp": {
+>                 "name": "Thermal Temp",
+>                 "type": "UINT8",
+>                 "normalized": true
+>               },
+>               "insulation": {
+>                 "name": "Insulation",
+>                 "type": "UINT8",
+>                 "normalized": true
+>               },
+>             }
+>           }
+>         }
+>       },
+>       "propertyTextures": [{
+>         "class": "thermalSample",
+>         "index": 0,
+>         "texCoord": 0,
+>         "properties": {
+>           "heatLoss": [0],
+>           "insulation": [1]
+>         }
+>       }]
+>     }
+>   }
+> }
+> ```
+>
+> _Primitive_
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "attributes": {
+>         "POSITION": 0,
+>         "TEXCOORD_0": 1
+>       },
+>       "indices": 2,
+>       "material": 0,
+>       "extensions": {
+>         "EXT_mesh_features": {
+>           "propertyTextures": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+
+A `propertyTexture` object extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` specifies a texture coordinate set in the referring primitive.
+
+The `properties` map specifies the texture channels providing data for available properties. An array of integer index values identifies channels, where multiple channels may be used only for fixed-length arrays of 2, 3, or 4 components. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. All values are stored in linear space.
+
+> **Example:** A property texture specifying that "thermalTemp" property values are stored in channel `R`, and "insulation" property values are stored in channel `G`, both indexed by UV coordinates in a `TEXCOORD_0` attribute.
```

Comment:
Should this be "channel 0" and "channel 1" instead of "channel `R`" and "channel `G`"?

Or alternatively:

> "thermalTemp" property values are stored in the red channel, and "insulation" property values are stored in the green channel

Reply:
I think spelling it out ("red", "green") is helpful as a complement to explain the numerical indices in the JSON below. ✅